src/modules/auth/server/procedures.ts

```
@@ -0,0 +1,151 @@
+import z from "zod"
```

ESLint

[error] 'z' is defined but never used.

(@typescript-eslint/no-unused-vars)